inductiva/structures/scenarios/plate_linear_elastic/bc_utils.py

```diff
@@ -0,0 +1,141 @@
+"""Utils to create the different types of boundary conditions."""
+
+from abc import ABC, abstractmethod
+from typing import Optional, List, Literal
+
+import json
+
+
+class BC(ABC):
+    """Abstract base class for boundary condition.
+
+    Attributes:
+        boundary_name (float): The boundary name of the plate where the boundary
```

Comment:
Wouldn't it be more natural to have a reference to the boundary, which in case would have a name? Isn't there a Boundary class (not saying there should be but is you have a BoundaryCondition class, it would not be suprising to have a Boundary class).

Reply:
I see your point, but will the boundary be named "boundary" exclusively? Adding the "Boundary" class would provide more structure to the code, but wouldn't people need to initialize the "Boundary" class to generate the boundary condition?